app/src/main/java/com/sopt/withsuhyeon/core/component/bottomsheet/BlockBottomSheet.kt

```diff
@@ -0,0 +1,139 @@
+package com.sopt.withsuhyeon.core.component.bottomsheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.Text
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.sopt.withsuhyeon.R
+import com.sopt.withsuhyeon.core.util.modifier.noRippleClickable
+import com.sopt.withsuhyeon.ui.theme.Grey500
+import com.sopt.withsuhyeon.ui.theme.WithSuhyeonTheme.colors
+import com.sopt.withsuhyeon.ui.theme.WithSuhyeonTheme.typography
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BlockBottomSheet(
+    modifier: Modifier = Modifier,
+    closeSheet: () -> Unit
+) {
+    val sheetState = rememberModalBottomSheetState()
+    ModalBottomSheet(
+        onDismissRequest = closeSheet,
+        sheetState = sheetState,
+        shape = RoundedCornerShape(topStart = 24.dp, topEnd = 24.dp),
+        containerColor = colors.Grey500,
+        dragHandle = null
+    ) {
+        Column(
+            modifier = modifier
+                .fillMaxWidth()
+                .background(colors.White)
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(start = 16.dp, top = 16.dp, end = 16.dp),
+                horizontalArrangement = Arrangement.spacedBy(10.dp, Alignment.End),
+
+                ) {
+                Icon(
+                    modifier = Modifier.noRippleClickable { },
+                    painter = painterResource(R.drawable.ic_xclose),
```

Comment:
벡터 리소스는 imageVector = ImageVector.vectorResource(R.drawable.ic_xclose)로 적어주실 수도 있습니다! 저도 예전에 코드리뷰 받았었는데 imageVector사용시 불필요한 리컴포지션을 방지해줄 수 있다고 합니다! 
```suggestion
                    imageVector = ImageVector.vectorResource(R.drawable.ic_xclose),
```